vm-setup/roles/firewall/tasks/iptables.yaml

```diff
@@ -13,6 +13,7 @@
     match: udp
     destination_port: "{{ vbmc_port_range }}"
     jump: ACCEPT
+    state: "{{ firewall_rule_state }}"
```

Comment:
Are you expecting `state: present` ?

Reply:
I got it, so when disabling rules you pass the `absent` value and when in opposite which is enabling you just rely on the default (`present`) value.

Reply:
Yes You got it